components/docker-up/runc-facade/main.go

```diff
@@ -38,7 +38,7 @@ func main() {
 	}
 
 	if useFacade {
-		err = createAndRunc(runcPath)
+		err = createAndRunc(runcPath, log)
```

Comment:
no need to pass in `log` here ... you can just use the functions from `common-go/log` directly.